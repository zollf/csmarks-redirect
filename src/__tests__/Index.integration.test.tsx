import React from 'react';
import { RenderResult, render } from '@testing-library/react';

import Index from '@/pages/index';

const component = <Index />;

describe('<Index />', () => {
  let wrapper!: RenderResult;
  beforeEach(() => {
    wrapper = render(component);
  });

  it('matches its snapshot', () => {
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
