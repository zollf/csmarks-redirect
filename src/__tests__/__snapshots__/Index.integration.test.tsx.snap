// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> matches its snapshot 1`] = `
<DocumentFragment>
  <div
    class="index"
  >
    <a
      href="https://secure.csse.uwa.edu.au/run/csmarks"
      rel="noopener noreferrer"
      target="_blank"
    >
      Click to open csmarks
    </a>
  </div>
</DocumentFragment>
`;
